fiaas_deploy_daemon/deployer/kubernetes/deployment/deployer.py

```diff
@@ -146,6 +151,10 @@ def _make_volumes(self, app_spec):
 
     def _make_volume_mounts(self, app_spec):
         volume_mounts = []
+        if app_spec.config_maps:
+            for config_map in app_spec.config_maps:
+                volume_mounts.insert(0, VolumeMount(name="{}-config".format(config_map),
+                                                    readOnly=True, mountPath="/var/run/config/fiaas/"))
```

Comment:
Configmaps are mounted as directories, where each key in the configmap becomes a file in the directory. They probably need distinct mount paths. Assuming the configmap name is a valid directory name (I think it would be), I suggest just mounting at `/var/run/config/<configmap-name>`.